src/components/InjectedComponents/AdditionalPostBox.tsx

```diff
@@ -7,16 +7,19 @@ import Services from '../../extension/service'
 import { geti18nString } from '../../utils/i18n'
 import { makeStyles } from '@material-ui/styles'
 import { Box, Button, Card, CardHeader, Divider, InputBase, Paper, Typography } from '@material-ui/core'
-import { Person, Group } from '../../database'
+import { Group, Person } from '../../database'
 import { NotSetupYetPrompt } from '../shared/NotSetupYetPrompt'
-import { useCurrentIdentity, useFriendsList, useMyIdentities, useGroupsList } from '../DataSource/useActivatedUI'
+import { useCurrentIdentity, useFriendsList, useGroupsList, useMyIdentities } from '../DataSource/useActivatedUI'
```

Comment:
Let us... not swap the imports back and forth anymore...

Reply:
emmmmm this was done by webstorm... if no more I need to clean unused import by myself...

Reply:
Not a serious problem now